web-ui/public/styles.css

```
@@ -1280,17 +1327,80 @@ body[data-screen="task"] .viewer-screen {
+}
+
+.execution-port-next {
+  right: 3px;
+}
+
+.execution-port-anchor {
+  flex: 0 0 auto;
+  width: 5px;
+  height: 5px;
+  box-sizing: border-box;
+  border: 1px solid currentColor;
```

Stylelint (17.14.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)